app/views/viewing_parties/new.html.erb

```diff
@@ -7,19 +7,22 @@
   <%= f.number_field :duration, options = { value: @movie.runtime, min: @movie.runtime } %>
 
   <%= f.label :date, 'Date' %>
-  <%= f.date_select :date, { use_two_digit_numbers: true, prefix: :start }%>
+  <%= f.date_field :date %>
```

Comment:
the only thing I could add to this would be to add `min: Date.Today` or something so that you can't create viewing parties for days that have already passed. I have the same critique for time value as well. Otherwise this looks way cleaner for the user